src/main/drivers/pwm_output.c

```diff
@@ -397,7 +395,9 @@ void pwmWriteDshotCommand(uint8_t index, uint8_t command)
         for (; repeats; repeats--) {
             motor->requestTelemetry = true;
             pwmWriteDshotInt(index, command);
-            pwmCompleteMotorUpdate(0);
+            if (pwmMotorsEnabled) {
```

Comment:
I think this is wrong, `pwmCompleteDshotMotorUpdate(0)` should be executed even when `pwmMotorsEnabled` is false, since CLI uses `pwmDisableMotors()` to stop motor updates while Dshot commands are sent.